src/components/Playlist/index.js

```diff
@@ -24,7 +25,67 @@ const CategoryIcon = ({icon}) => {
   return <img src={icon} className='ml2 mt1 h1' alt='' />
 }
 
-const PlaylistItem = ({item, extraClasses}) => {
+const StyledPlaylistItem = styled.li`
+	&:before, &:after {
+    content: '';
+    position: absolute;
+    transition: 150ms;
+	}
+	&:before {
+    width: 9px;
+    height: 9px;
+    border-radius: 50%;
+    border: 1px solid var(--light-gray);
+    background: var(--tag-gray);
+    box-shadow: 0 0 0 1px var(--tag-gray);
+    left: 20px;
+    top: 22px;
+    z-index: 2;
+	}
+	&:after {
+    width: 1px;
+    height: 100%;
+    border-width: 0 0 0 1px;
+    border-style: solid;
+    border-color: var(--light-gray);
+    left: 24px;
+    top: 32px;
+    z-index: 1;
+	}
+	&:hover:before {
+    background: var(--white);
+    box-shadow: 0 0 0 1px var(--white);
+	}
+	&.viewed {
+    color: var(--gray);
+	}
+  &.viewed:before {
+    border: 1px solid ${props => props.card === 'playlist' ? 'var(--turquoise)' : 'var(--orange)'};
+    background: ${props => props.card === 'playlist' ? 'var(--turquoise)' : 'var(--orange)'};
```

Comment:
Example of using string interpolation to choose a CSS value based on a prop's value, same as how it would work for just a `className`.